addons/l10n_cl/models/res_partner.py

```diff
@@ -8,9 +8,15 @@ class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = 'res.partner'
 
+    def _get_default_country_id(self):
+        allowed_company = self._context.get('allowed_company_ids')
+        if not allowed_company:
+            return False
+        company = self.env['res.company'].browse(allowed_company[0])
+        return company.country_id == self.env.ref('base.cl') and self.env.ref('base.cl')
```

Comment:
We should not add such things in the localization.  If we do it, it should be in the standard and in the standard, the user can easily add a default value himself if he wants to.  We took these cases into account in the l10n_latam_base module.

Reply:
Removed here -> 535d8bf